source/community/release_notes/palm.rst

```diff
@@ -15,14 +15,109 @@ These are the release notes for the Palm release, the 16th community release of
 Breaking Changes
 ****************
 
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
+
+Stripe API changes
+==================
+
+Ecommerce now supports the new Stripe Payment Intents API and is no longer using the Stripe Charges API. The legacy 
+Stripe Charges ecommerce payment processor with a frontend in the ecommerce application is upgraded to a Stripe 
+Payment Intents payment processor with a frontend in frontend-app-payment. See the discussion post, `Ecommerce: 
+Changes to Stripe payment processor 
+<https://discuss.openedx.org/t/ecommerce-changes-to-stripe-payment-processor/9457>`_ for more details. 
+
+Ecommerce is still officially deprecated and we continue to work on the `Ecommerce Replacement Project 
+<https://openedx.atlassian.net/wiki/spaces/AC/pages/3617849345/Ecommerce+Deprecation+and+Replacement+Project>`_.
+
+
+Developer changes
+=================
+
+The old developer documentation site has been deprecated and removed. Please use `Open edX Developers - Latest 
+Documentation <https://docs.openedx.org/en/latest/developers/index.html>`_  instead.
+
+part of the Tutor API. This change should not affect most developers, who only use the :code:`Actions` and :code:`Filters` 
+classes (notice the plural) from :code:`tutor.hooks`. If it does affect you, there are upgrade instructions in the 
+discussion post, `Simplifying & documenting the Tutor Hooks API 
+<https://discuss.openedx.org/t/simplifying-documenting-the-tutor-hooks-api/9258>`_
+
+Most of the viewing code for PDF certificates was removed in previous releases. In the Palm release, we closed out 
+this work with an ADR (Application Decision Record) on `Leaving PDF Certificate Fields in Certificates Model 
+<https://github.com/openedx/edx-platform/blob/open-release/palm.master/lms/djangoapps/certificates/docs/decisions/008-certificate-model-remnants.rst>`_. 
+
 
 Learner Experiences
 *******************
 
+New Learner Home Page 
+=====================
+
+The new Learner Home has many of the same features of the old dashboard, with some extended functionality and 
+performance enhancements.
+
+* The Learner Home is now built with Paragon, the open edX design pattern library. It is accessible and easy to style with brand colors. 
+* Course cards show the course thumbnail, information about the course, and the ability to upgrade to a paid track or view/begin a course (if applicable). Further course actions (e.g. unenroll, email opt-out settings, and social media share) have been moved to the menu/triple dot icon on the course card.
```

Comment:
Can you add an image of the new course card here? I find the image of the Refine pop up confusing without seeing the new course card to begin with.